### PR TITLE
Prepare release 1.6.0

### LINKED ISSUE #919: Version 1.6 countdown
I would like to include the following PRs before we release 1.6:

* [x] Include HotROD binary in the distributions #917 
* [x] Improve HotROD demo #915
* [x] Package static assets inside the query-service binary #918
* [x] Update Dockerfiles to reference executable via ENTRYPOINT #815 
* [x] ES 6.x index race condition https://github.com/jaegertracing/jaeger/pull/921

### PR DESCRIPTION
**Pull request by yurishkuro**
Closes #919

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,64 @@
...
+- Ability to use multiple storage types (#880) by David Yeghshatyan (@davit-y)
+
+##### Minor Improvements
+
+- [ES storage] Log number of total and failed requests (#902) by Tomasz Adamski (@tmszdmsk)
+- [ES storage] Do not log requests on error (#901) by Tomasz Adamski (@tmszdmsk)
+- Add cassandra index filter (#876) by Won Jun Jang (@black-adder)
+- Close span writer in standalone (#863) (4 weeks ago) by Pavol Loffay (@pavolloffay)
+- Log configuration options for memory storage (#852) (6 weeks ago) by Juraci Paixão Kröhling (@jpkrohling)
+- Update collector metric counters to have a name (#886) by Won Jun Jang (@black-adder)
```
Comment (yurishkuro) [inline CHANGELOG.md]: @black-adder was this a breaking change? Not clear from PR description.

Comment (black-adder) [reply CHANGELOG.md]: it was not, pre and post change should result in the same metric name

**Comment by black-adder** [comment]
@yurishkuro can we hold off on releasing this? @tiffon wants to land the new ui changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,64 @@
...
+  run
+  ```
+  $ docker run -it --rm jaegertracing/jaeger-collector help
+  ```
+
+- Detect HTTP payload format from Content-Type (#916) by Yuri Shkuro (@yurishkuro)
+
+  When submitting spans in Thrift format to HTTP endpoint `/api/traces`,
+  the `format` argument is no longer required, but the Content-Type header
+  must be set to "application/vnd.apache.thrift.binary".
```
Comment (black-adder) [inline CHANGELOG.md]: or `application/x-thrift`?

**Comment by yurishkuro** [comment]
> Should we also publish a new blog post, with a content similar to this changelog?

@jpkrohling if you're up to it, sure. And then tweet.

**Pull request merged**